cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -1732,15 +1739,90 @@ def start(self):
             service.verify_alive(POSTGRES_SERVICE_NAME)
         logger.notice('PostgreSQL Server successfully started')
 
+    def _save_encoding_and_locale(self):
+        self.encoding = subprocess.run(
+            ['psql', '-U', 'postgres', '-c', 'SHOW SERVER_ENCODING', '-t'],
+            stdout=subprocess.PIPE, stderr=subprocess.DEVNULL
+        ).stdout.decode('utf-8').strip()
+
+        self.locale = subprocess.run(
+            ['psql', '-U', 'postgres', '-c', 'SHOW LC_CTYPE', '-t'],
+            stdout=subprocess.PIPE, stderr=subprocess.DEVNULL
+        ).stdout.decode('utf-8').strip()
```

Comment:
I don't see the need for these two calls specifically to be `subprocess.run` rather than `common.run` (like the rest of them)

Reply:
also I wonder how come those aren't just... known. We never change them, surely they are just constants

Reply:
alas they aren't constants